src/main/resources/db/changelog/changesets/changelog_20230817T145909Z.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="labidiayo (generated)" id="1692284360171-1">
+        <createTable tableName="volatge_limits_parameter">
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="volatge_limits_parameterPK"/>
+            </column>
+            <column name="high_voltage_limit" type="FLOAT8"/>
+            <column name="low_voltage_limit" type="FLOAT8"/>
+            <column name="priority" type="INT"/>
+            <column name="voltage_init_setting_id" type="UUID"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-2">
+        <createTable tableName="voltage_init_setting">
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="voltage_init_settingPK"/>
+            </column>
+            <column name="date" type="TIMESTAMP"/>
+            <column name="name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-3">
+        <createTable tableName="voltage_init_setting_entity_constantqgenerators">
+            <column name="voltage_init_setting_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-4">
+        <createTable tableName="voltage_init_setting_entity_variable_shunt_compensators">
+            <column name="voltage_init_setting_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-5">
+        <createTable tableName="voltage_init_setting_entity_variable_two_wt">
```

Comment:
It's nitpicky but we could we could take advantage of this migration to rename `two_wt` to `twt` which is a more widely denomination used across our apps

Reply:
or maybe `twt` could become an issue someday when we expand three windings transformers implementation